pnpm-lock.yaml

```diff
@@ -28,100 +28,100 @@ importers:
     devDependencies:
       '@commitlint/cli':
         specifier: ^17.7.1
-        version: 17.7.1
+        version: registry.npmmirror.com/@commitlint/cli@17.7.1
```

Comment:
Remove the npmmirror.

Reply:
https://pnpm.io/npmrc#https-proxy
https://pnpm.io/npmrc#http-proxy

Reply:
why should we modify pnpm-lock.yaml manuallly? it's generated

Reply:
You should not modify it manually. What you need to do is to use the proxy to pull from npmjs rather than the mirror.

Reply:
For oversea users, the npmmirror field in the pnpm lock file will force the pnpm to request the package from the npmmirror. That will slow down their installation process (including CI), and may violate their goverment or company policies.

Reply:
okay, it's because in my machine I use npm mirror. 
It would be fine that I use the official npm source.